Editor/SettingsAnalysis/QualitySettingsAnalyzer.cs

```diff
@@ -152,5 +182,49 @@ internal static bool IsDefaultAsyncUploadBufferSize()
             QualitySettings.SetQualityLevel(initialQualityLevel);
             return usingDefaultAsyncUploadBufferSize;
         }
+
+        internal static List<int> GetTextureStreamingDisabledQualityLevelsIndex()
+        {
+            var initialQualityLevel = QualitySettings.GetQualityLevel();
+            var qualityIndexes = new List<int>();
+
+            for (var i = 0; i < QualitySettings.names.Length; i++)
+            {
+                QualitySettings.SetQualityLevel(i);
+
+                if (!QualitySettings.streamingMipmapsActive)
+                {
+                    qualityIndexes.Add(i);
+                }
+            }
+
+            QualitySettings.SetQualityLevel(initialQualityLevel);
+            return qualityIndexes;
+        }
+
+        static void EnableStreamingMipmap(int pQualityLevelIndex)
```

Comment:
Code style, this is maybe just a typo:

Instead of pQualityLevelIndex I would just write "qualityLevelIndex", I mean without the additional "p" prefix.